aws_lambda_builders/workflows/python_pip/actions.py

```diff
@@ -3,25 +3,37 @@
 """
 
 from aws_lambda_builders.actions import BaseAction, Purpose, ActionFailedError
-from .packager import PythonPipDependencyBuilder, PackagerError
+from aws_lambda_builders.workflows.python_pip.utils import OSUtils
+from .packager import PythonPipDependencyBuilder, PackagerError, DependencyBuilder, SubprocessPip, PipRunner
 
 
 class PythonPipBuildAction(BaseAction):
 
     NAME = 'ResolveDependencies'
     DESCRIPTION = "Installing dependencies from PIP"
     PURPOSE = Purpose.RESOLVE_DEPENDENCIES
+    LANGUAGE = 'python'
 
-    def __init__(self, artifacts_dir, manifest_path, scratch_dir, runtime):
+    def __init__(self, artifacts_dir, manifest_path, scratch_dir, runtime, binaries):
         self.artifacts_dir = artifacts_dir
         self.manifest_path = manifest_path
         self.scratch_dir = scratch_dir
         self.runtime = runtime
-        self.package_builder = PythonPipDependencyBuilder(runtime=runtime)
+        self.binaries = binaries
 
     def execute(self):
+        os_utils = OSUtils()
+        python_path = [binary.binary_path for binary in self.binaries if getattr(binary, self.LANGUAGE)][0]
```

Comment:
Not sure I understand this line of code. Could you explain this? My brain can't grasp this right now

Reply:
yes.
if binary_path.python is set, we get the associated binary_path, this is why I do setattr(self, binary, binary) above in the class.

so that binary_path.python is set.